ArraysHomework/ArraysHomework.cs

```diff
@@ -51,10 +63,13 @@ public void Arrays()
 
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
-            
+            Array.Reverse(swapMyValues);
```

Comment:
What if there were more than three elements in the array? `Reverse` would change more than just the first and last elements.